api/metric/doc.go

```diff
@@ -60,10 +60,11 @@
 // the New*Measure function - this allows reporting negative values
 // too. To report a new value, use the Record function.
 //
-// All the basic kinds of instruments also support creating handles
-// for a potentially more efficient reporting. The handles have the
-// same function names as the instruments (so counter handle has Add,
-// gauge handle has Set and measure handle has Record). Handles can be
-// created with the AcquireHandle function of the respective
-// instrument. When done with the handle, call Release on it.
+// All the basic kinds of instruments also support creating bound
+// instruments for a potentially more efficient reporting. The bound instruments
+// have the same function names as the instruments (so counter boundInstrument
+// has Add, gauge boundInstrument has Set and measure boundInstrument has Record).
```

Comment:
Please change "boundInstrument" to "bound instrument" in sentence contexts like this.